plone/app/contenttypes/browser/templates/listing.pt

```diff
@@ -93,7 +93,7 @@
 
                               <tal:event condition="python:item_is_event">
                                 <tal:date tal:replace="structure python:view.formatted_date(obj)" />
-                                <span tal:condition="python:obj.location"
+                                <span tal:condition="python:obj.location if hasattr(obj, 'location') else None"
```

Comment:
Isn't 
```suggestion
                                <span tal:condition="python:getattr(obj, 'location', None)"
```
a bit simpler?

Reply:
yes, much simpler. Should we take care of acquisition also? something like `getattr(obj.aq_base, 'location', None)`?